backend/infrahub/graphql/queries/account.py

```diff
@@ -120,17 +119,13 @@ async def resolve_account_permissions(
         raise ValueError("An account_session is mandatory to execute this query")
 
     fields = await extract_fields_first_node(info)
-    permissions: AssignedPermissions = {"global_permissions": [], "object_permissions": []}
-    for permission_backend in registry.permission_backends:
-        backend_permissions = await permission_backend.load_permissions(
-            db=context.db, account_session=context.account_session, branch=context.branch
-        )
-        permissions["global_permissions"].extend(backend_permissions["global_permissions"])
-        permissions["object_permissions"].extend(backend_permissions["object_permissions"])
+
+    permission_manager = PermissionManager(account_session=context.account_session)
```

Comment:
same here, could we retrieve it from the context `context: GraphqlContext = info.context`